### DISCUSSION TITLE
Know Hosts Error

### DISCUSSION THREAD

**Discussion by WillyReyno** [Sequel-Ace/Sequel-Ace#1215] [Q&A]
Hello, 

I'm using Sequel Ace to login into my VM's MySQL Docker container using SSH. The connection works, however at each SequelAce startup I'm getting a permission issue. 

Here is my SSH configuration : 
<img width="458" alt="ssh-config" src="https://user-images.githubusercontent.com/5576482/131715173-c1f4306c-0ea7-440c-991e-1f3ff2499778.png">

Here is the error I get : 
<img width="286" alt="error" src="https://user-images.githubusercontent.com/5576482/131715168-994e49ef-2e55-4e78-9d37-c8542972fa38.png">

I have a workaround which is to click on the "key" icon on the SSH Key line and selecting my `id_rsa` key file again which makes the message disappear and let me login. 

However, if I quit Sequel-Ace and reopen it, the error happens again. 

Has anyone encountered the same issue and might have an idea why this is happening?

**Comment by Jason-Morcos** [comment]
If you go to the files section in Sequel Ace's prefs and remove that file from the list, restart the app, and then manually select the key again in the connection settings, does that fix the issue?

Or, is there any chance something else is _modifying_ the id_rsa, thereby requiring Ace to re-request access? I would take a look at the last modified date for the key in Finder.

**Reply by WillyReyno** [reply]
Hello, thanks for your answer! 

I tried what you asked, but the issue remains.

I checked the last modified date of the id_rsa, and it is June 18th, so it doesn't seem to be modified by something...

**Reply by Jason-Morcos** [reply]
Very odd! You should be able to work around this issue by giving Sequel Ace "Full Disk Access" in System Preferences, but you shouldn't have to do that. My thought is somehow your permissions on your computer are corrupted and are causing these odd symptoms.

**Reply by WillyReyno** [reply]
Unfortunately, even giving Sequel Ace full disk access didn't fix it 😅

**Reply by Jason-Morcos** [reply]
I think the next thing to try would be resetting all rivaux settings on your Mac! Still think something's corrupted here... https://www.techrepublic.com/article/how-to-reset-a-users-privacy-permissions-on-macos-and-ios/

That would make all privacy prompts show again for all apps though :/
